drivers/hmis/app/models/hmis/user.rb

```diff
@@ -60,12 +60,17 @@ def load_effective_permissions
     define_method("#{permission}_for?") do |entity|
       return false unless send("#{permission}?")
 
+      # Use global permission scope if client is unenrolled
+      return send("#{permission}?") if entity.is_a?(Hmis::Hud::Client) && entity.projects.empty?
```

Comment:
Maybe a more descriptive comment: `# Use global permission scope if entity is a Client, and Client does not have any non-WIP Enrollments`

@eanders what do you think about whether to take WIP Enrollments into consideration when determining whether a client is "unenrolled" or not? I think if that enrollment has been started, the client access should probably be limited right away. 

@ZachCMP I didn't think through allll the implications when you asked in PT, sorry this parts a bit tricky. Clients that only have WIP enrollments are considered "unenrolled" for _reporting_ purposes, but for access purposes I think we should consider them enrolled. See https://github.com/greenriver/hmis-warehouse/pull/3004 for how this could be updated to use `projects_including_wip` if so. Lets wait to hear what Elliot thinks though.

Reply:
Agreed. WIP should be treated as an enrollment for access purposes.

Reply:
Okay, so we **should** count clients as enrolled as soon as they have an enrollment, even if it's incomplete and the intake assessment is in progress, correct?

Reply:
Yes